systemd/ocp-cluster-ca.sh

```diff
@@ -11,72 +11,51 @@ export KUBECONFIG="/opt/kubeconfig"
 
 wait_for_resource configmap
 
-custom_ca_path=/opt/crc/custom-ca.crt
 external_ip_path=/opt/crc/eip
 
-if [ ! -f ${custom_ca_path} ]; then
-    echo "Cert bundle /opt/crc/custom-ca.crt not found, exiting..."
-    exit 1
-fi
-
 if [ ! -f /opt/crc/pass_kubeadmin ]; then
     echo "kubeadmin password file not found"
     exit 1
 fi
 
-# base64 decode the cert
-base64 -d < "${custom_ca_path}" > /tmp/custom-kubeadmin-ca.crt
-custom_ca_path=/tmp/custom-kubeadmin-ca.crt
+# generate CA
```

Comment:
We should mention as comment that those steps are part of https://access.redhat.com/solutions/6054981 so that in future, it can be referred

Reply:
there's a comment listing the links at the top of the file https://github.com/crc-org/snc/pull/1088/files#diff-57ea196af424972bc0595d747b0f0a60be2f3ac197b41b1b357a6a173cde7abaL4-L5